src/aig-common/modules/iam/common-iam.module.ts

```diff
@@ -31,29 +31,28 @@ import { AigRoleAssociationTableComponent } from './components/role-association-
 import { AigAutocompleteFilterService } from './services/form/autocomplete-filter.service';
 import { AigAutocompleteFunctionService } from './services/form/autocomplete-function.service';
 import { AigValidatorService } from './services/form/validator.service';
-import { RouterModule } from '@angular/router';
 
 @NgModule({
     imports: [
         AigCommonModule,
+
         CommonModule,
 
         BrowserModule,
         FormsModule,
         ReactiveFormsModule,
 
-        FlexLayoutModule,
-        RouterModule,
-
         MatTableModule,
         MatProgressSpinnerModule,
+
         MatButtonModule,
         MatIconModule,
+        
         MatInputModule,
         MatFormFieldModule,
         MatAutocompleteModule,
 
-        
+        FlexLayoutModule,
 
     ],
     declarations: [
```

Comment:
restore original of this file

Reply:
this is the original:
[https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/iam/common-iam.module.ts](url)